Formula/l/lv.rb

```diff
@@ -1,11 +1,12 @@
 class Lv < Formula
   desc "Powerful multi-lingual file viewer/grep"
-  homepage "https://web.archive.org/web/20160310122517/www.ff.iij4u.or.jp/~nrt/lv/"
-  url "https://web.archive.org/web/20150915000000/www.ff.iij4u.or.jp/~nrt/freeware/lv451.tar.gz"
-  version "4.51"
-  sha256 "e1cd2e27109fbdbc6d435f2c3a99c8a6ef2898941f5d2f7bacf0c1ad70158bcf"
+  # The upstream homepage was "https://web.archive.org/web/20160310122517/www.ff.iij4u.or.jp/~nrt/lv/"
+  homepage "https://salsa.debian.org/debian/lv"
+  url "https://salsa.debian.org/debian/lv/-/archive/debian/4.51-9/lv-debian-4.51-9.tar.gz"
+  version "4.51-9"
```

Comment:
`brew audit --strict lv` passes locally, but the CI is failing with `Stable: version 4.51-9 is redundant with version scanned from URL`